### ISSUE TITLE
Reading：How Many Data Points is a Prompt Worth?

### ISSUE THREAD

**Issue by tikiti-kyotaro** [tikiti-kyotaro/Paperlog#4]
[Teven Le Scao](https://arxiv.org/search/cs?searchtype=author&query=Scao%2C+T+L), [Alexander M. Rush](https://arxiv.org/search/cs?searchtype=author&query=Rush%2C+A+M)

> When fine-tuning pretrained models for classification, researchers either use a generic model head or a task-specific prompt for prediction. Proponents of prompting have argued that prompts provide a method for injecting task-specific guidance, which is beneficial in low-data regimes. We aim to quantify this benefit through rigorous testing of prompts in a fair setting: comparing prompted and head-based fine-tuning in equal conditions across many tasks and data sizes. By controlling for many sources of advantage, we find that prompting does indeed provide a benefit, and that this benefit can be quantified per task. Results show that prompting is often worth 100s of data points on average across classification tasks.

arXive : https://arxiv.org/abs/2103.08493

# 1. どんなもの？
- promptがどれくらいのデータと同じ価値を持っているのかということを実験した

# 2. 先行研究と比べてどこがすごい？
- 各プロンプトの価値を定量的に検証することができている

# 3. 技術や手法のキモはどこ？
- 一番上がpromptの価値を表している
<img width="756" alt="スクリーンショット 2022-10-21 13 17 43" src="https://user-images.githubusercontent.com/88369097/197111209-b44af110-35c1-436a-ae1e-e395acc84c06.png">

# 4. どうやって有効だと検証した？
- SuperGLUEのさまざまなデータセットを使ってhead-basedとprompt-basedのtuningをした
	- 10データポイントから徐々に増やしていく

# 5. 議論はある？

# 6. 次に読むべき論文は？

#2022 #arXiv
